iree/test/e2e/regression/BUILD

```diff
@@ -221,3 +225,31 @@ py_binary(
 ] for size in [
     "small",
 ]]
+
+[iree_generated_trace_runner_test(
+    name = "e2e_matmul_direct_f32_gpu_large_%s" % compilation_info,
+    generator = ":generate_e2e_matmul_tests",
+    generator_args = [
+        "--lhs_rhs_type=f32",
+        "--shapes=gpu_large",
```

Comment:
I made the "large" shapes a lot less large and these test should run substantially faster now.

Reply:
FWIW the existing e2e matmul tests for CPU are a lot more expensive (test many more shapes) so if this is the concern, we can find more opportunity for improvement on that side.

Reply:
Actually i had been meaning to do that , so i'll do it now - will report back here.
Anyway, it can't get cheaper than what this PR does for GPU -- just one shape, and not even particularly large either.

Reply:
https://github.com/google/iree/pull/8448